pkg/api/validation/validation.go

```diff
@@ -2754,10 +2759,22 @@ func ValidateResourceRequirements(requirements *api.ResourceRequirements, fldPat
 		if api.IsStandardResourceName(string(resourceName)) {
 			allErrs = append(allErrs, validateBasicResource(quantity, fldPath.Key(string(resourceName)))...)
 		}
+		if resourceName == api.ResourceMemory && !IsResourceRequirementsMemoryValid(quantity) {
+			allErrs = append(allErrs, field.Invalid(fldPath, quantity.String(), "memory request must not be configured in millibytes"))
+		}
 	}
 	return allErrs
 }
 
+// validateResourceRequirementsMemory ensures that memory is not configured in milli value
+func IsResourceRequirementsMemoryValid(quantity resource.Quantity) bool {
+	memStr := quantity.String()
+	if memStr[len(memStr)-1] == 'm' {
```

Comment:
agreed... i had a similar need when working on https://github.com/openshift/origin/pull/8409/files#diff-5fd1547857aea1ef0cdb18cb293f88faR162  but punted on at it at the time.  you can see the same rounding logic there though.